src/sif/dynamiccost.cc

```diff
@@ -146,9 +146,9 @@ DynamicCost::DynamicCost(const Costing& costing,
                          const TravelMode mode,
                          uint32_t access_mask,
                          bool penalize_uturns)
-    : pass_(0), allow_transit_connections_(false), allow_destination_only_(true),
-      allow_conditional_destination_(false), travel_mode_(mode), access_mask_(access_mask),
-      closure_factor_(kDefaultClosureFactor), flow_mask_(kDefaultFlowMask),
+    : pass_(0), allow_transit_connections_(false), destination_only_pruning_(true),
```

Comment:
renamed the members to be consistent (now it has the opposite meaning of before, so I had to change all logic where this is used or set)